src/showcase/nz-demo-upload/nz-demo-upload-basic.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'nz-demo-upload-basic',
+  template: `
+  <nz-upload
+    nzAction="https://jsonplaceholder.typicode.com/posts/">
+    <button nz-button>
+      <span><i class="anticon anticon-upload"></i>Click to Upload</span>
```

Comment:
should be <i class="anticon anticon-upload"></i> and <span>Click to Upload</span>